ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-rhoai.yaml

```diff
@@ -55,6 +55,8 @@ tests:
     allow_best_effort_post_steps: true
     allow_skip_on_success: true
     cluster_profile: gcp-opendatahub
+    dependencies:
+      OO_INDEX: opendatahub-operator-rhoai-bundle
```

Comment:
OO_BUNDLE ?

Reply:
do i understand it for now:
- odh main, we build with bundle+srccat
-  odh rhoai, we build with bundle+fbc

but in about 2 months, when we shift odh main to konflux, that need to be switch to
- odh main and odh rhoai, both bundle+fbc